lua/utils/init.lua

```diff
@@ -99,55 +99,10 @@ function utils.reload_lv_config()
   vim.cmd ":PackerCompile"
   vim.cmd ":PackerInstall"
   -- vim.cmd ":PackerClean"
-  Log:get_default().info "Reloaded configuration"
-end
-
-function utils.check_lsp_client_active(name)
-  local clients = vim.lsp.get_active_clients()
-  for _, client in pairs(clients) do
-    if client.name == name then
-      return true
-    end
-  end
-  return false
-end
-
-function utils.get_active_client_by_ft(filetype)
-  local clients = vim.lsp.get_active_clients()
-  for _, client in pairs(clients) do
-    if client.name == lvim.lang[filetype].lsp.provider then
-      return client
-    end
-  end
-  return nil
-end
+  local null_ls = require "lsp.null-ls"
+  null_ls.setup(vim.bo.filetype, { force_reload = true })
```

Comment:
Is this necessary? I feel like it might slow down the reload unintentionally, or is it about hot-reloading the configs?

Reply:
Yep, this is done to hot reload providers, it works visually (since galaxyline shows them), but I'm not sure null-ls handles it (yet ?)